lib/pbench/server/api/resources/pbench_users.py

```diff
@@ -0,0 +1,426 @@
+import jwt
```

Comment:
Can we rename this module from `pbench_users.py` to just `users.py`?

Reply:
Modified it to `users_api.py`, `users.py` is used for user database model